src/charts/utilization-bar/utilization-bar-chart.component.js

```diff
@@ -172,8 +172,16 @@ angular.module('patternfly.charts').component('pfUtilizationBarChart', {
       prevChartData = angular.copy(ctrl.chartData);
       prevLayout = angular.copy(ctrl.layout);
 
+      if (!ctrl.chartData) {
```

Comment:
Why are there changes in `utilization-bar-chart.component.js`?  Is this a git/rebase/commit issue?